### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][NVDA][MAS4.2.1][IntelliJ][Wipro]: NVDA is announicng extra information after invoking "Add" button using shortcut key

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1491]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. Create a new project.
3. Tab to and select created Project ->Azure->Run on web app.
4. "Run on web app" dialog will open.
5. Tab to and select Use Existing web app/ Create New Web App radio button radio button.
6. Navigate to  "Add" button  then press "Alt+Insert" then observe the NVDA Behavior

**Expected Result:**
**NVDA Behavior:**
NVDA should announce
Add New Configuration  list
Run External tool  selected  1 of 7

**Actual Result:**
**NVDA Behavior:**
NVDA is announcing as 
sample - IntelliJ IDEA
Run on Web App  dialog  Artifact
Please check the web.config file used to deploy this non-WAR executable.
Before launch: Build Artifacts, Activate tool window
Add New Configuration  list
Run External tool  selected  1 of 7
**Jaws Behavior:** Jaws is announcing as expected
Add (Alt+Insert) Button Pressed
To activate press spacebar.
Add New Configuration List view
Run External tool
1 of 7
To move to items use the Arrow keys.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jet brains.
OSBuild Version: Version 1709(Os Build 16299.192)
![run on web app-alt-insert](https://user-images.githubusercontent.com/22956130/36135709-ddb55ed4-10b2-11e8-8603-10d1058403c9.jpg)
![once user hits the add button by using shortcut key nvda announces some extra information in run on web app window](https://user-images.githubusercontent.com/22956130/36135746-0b27a94e-10b3-11e8-8842-4969c211ba30.jpg)

**Comment by Eskibear** [comment]
duplicate with #1500

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749953
